utils/config/daos_agent.yml

```diff
@@ -26,26 +26,35 @@
 # default: 10001
 #port: 10001
 
-## Enable HTTP endpoint for remote telemetry collection.
-# Note that enabling the endpoint automatically enables
-# client telemetry collection.
+## Enable Telemetry HTTP/HTTPS endpoint for remote client telemetry collection.
 #
-## default endpoint state: disabled
-## default endpoint port: 9192
-#telemetry_port: 9192
-
-## Enable client telemetry for all DAOS clients.
-# If false, clients will need to optionally enable telemetry by setting
-# the D_CLIENT_METRICS_ENABLE environment variable to true.
+#telemetry_config:
+#  # Set the client telemetry endpoint port number
+#  # default: 9192
+#  port: 9192
 #
-## default: false
-#telemetry_enabled: true
-
-## Retain client telemetry for a period of time after the client
-# process exits.
+#  # Enable client telemetry for all DAOS clients.
+#  # If false, clients will need to optionally enable telemetry by setting
+#  # the D_CLIENT_METRICS_ENABLE environment variable to true.
+#  # default: false
+#  enabled: true
+#
+#  # Retain client telemetry for a period of time after the client
+#  # process exits.
+#  # default 0 (do not retain telemetry after client exit)
+#  retain: 1m
+#
+#  # In order to disable transport security, uncomment and set allow_insecure
+#  # to true. Not recommended for production configurations.
+#  allow_insecure: false
```

Comment:
I think this will also changes once we use the tool suggested by you. So let me try and see how that works and will change the code accordingly.

Reply:
Code was modified to use the system certificate. But to be aligned with Transport config option I think it's better we can keep it. This will also gives user a chance to use specific security level with certificates.